hoomd/Integrator.h

```diff
@@ -138,6 +139,7 @@ class PYBIND11_EXPORT Integrator : public Updater
     protected:
     /// The step size
     Scalar m_deltaT;
+    std::shared_ptr<Trigger> m_trigger = std::make_shared<PeriodicTrigger>(1, 0);
```

Comment:
This isn't necessary, `Updater` already has an `m_trigger` member, and the `Integrator` constructor already passes a dummy trigger to the `Updater` base class.

Reply:
Ah, I missed that.